server/src/internalClusterTest/java/org/opensearch/cluster/routing/allocation/decider/DiskThresholdDeciderIT.java

```diff
@@ -465,6 +458,10 @@ private long createReasonableSizedShards(final String indexName) throws Interrup
         }
     }
 
+    private static FsInfo.Path setDiskUsage(FsInfo.Path original, long totalBytes, long freeBytes) {
+        return new FsInfo.Path(original.getPath(), original.getMount(), totalBytes, freeBytes, freeBytes);
```

Comment:
Nitpick, but you could probably just inline this as opposed to having a separate function. Also `setDiskUsage` is slightly misleading because this method just creates an instance and doesn't actually do the setting.